specification/eventgrid/data-plane/Microsoft.ResourceNotifications/stable/2018-01-01/common.json

```diff
@@ -0,0 +1,105 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2018-01-01",
+    "title": "Schema of the common properties for Azure Resource Notifications (ARN) System topics",
+    "description": "Describes the schema of the the common properties across all topic types published by ARN to Azure Event Grid. This corresponds to the Data property of an EventGridEvent."
+  },
+  "paths": {},
+  "definitions": {
+    "ResourceNotificationsEmittedEventData": {
+      "description": "Describes the schema of the common properties across all ARN system topic events",
+      "type": "object",
+      "properties": {
+        "resourceInfo": {
+          "description": "details of resourceInfo",
+          "type": "object",
+          "properties": {
+            "id": {
+              "description": "id of the resource for which the event is being emitted",
+              "type": "string"
+            },
+            "name": {
+              "description": "name of the resource for which the event is being emitted",
+              "type": "string"
+            },
+            "type": {
+              "description": "the type of the resource for which the event is being emitted",
+              "type": "string"
+            },
+            "location": {
+              "description": "the location of the resource for which the event is being emitted",
+              "type": "string"
+            },
+            "tags": {
+              "description": "the tags on the resource for which the event is being emitted",
+              "type": "string"
+            },
+            "properties": {
+              "description": "properties in the payload of the resource for which the event is being emitted",
+              "type": "object",
+              "additionalProperties": {
+                "description": "properties in the payload of the resource for which the event is being emitted"
```

Comment:
Type still needs to be specified, but instead of using `object` we can do this:
"additionalProperties": {}